Source: kde-packager@
Upstream: Yes, will be in 4.5.2.
Reason: Don't always disable krossruby.

From 2203e637b640223205b84e0fe217389bdabfac04 Mon Sep 17 00:00:00 2001
From: sitter <sitter@283d02a7-25f6-0310-bc7c-ecb5cbfe19da>
Date: Tue, 31 Aug 2010 10:22:33 +0000
Subject: [PATCH] Backport r1170253

Use proper cmake syntax (EQUAL, LESS...) when checking whether to build krossruby


git-svn-id: svn+ssh://svn.kde.org/home/kde/branches/KDE/4.5/kdebindings@1170254 283d02a7-25f6-0310-bc7c-ecb5cbfe19da
---
 ruby/CMakeLists.txt |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ruby/CMakeLists.txt b/ruby/CMakeLists.txt
index fd0bb25..32ab964 100644
--- a/ruby/CMakeLists.txt
+++ b/ruby/CMakeLists.txt
@@ -74,9 +74,9 @@ if (RUBY_EXECUTABLE AND RUBY_LIBRARY AND RUBY_INCLUDE_DIRS)
             #include <ruby.h>
             int main() { ruby_init(); return 0; }" RubyCompiles)
         if(RubyCompiles)
-            if(ENABLE_KROSSRUBY AND RUBY_VERSION_MAJOR = 1 AND RUBY_VERSION_MINOR < 9)
+            if(ENABLE_KROSSRUBY AND RUBY_VERSION_MAJOR EQUAL 1 AND RUBY_VERSION_MINOR LESS 9)
                 set(BUILD_KrossRuby TRUE)
-            endif(ENABLE_KROSSRUBY AND RUBY_VERSION_MAJOR = 1 AND RUBY_VERSION_MINOR < 9)
+            endif(ENABLE_KROSSRUBY AND RUBY_VERSION_MAJOR EQUAL 1 AND RUBY_VERSION_MINOR LESS 9)
         else(RubyCompiles)
             message(STATUS "Failed to compile a C++ program that embeds ruby.")
         endif(RubyCompiles)
-- 
1.7.2.2

